web_src/less/_base.less

```diff
@@ -1300,3 +1287,25 @@ i.icon.centerlock {
 .ui.sub.header {
     text-transform: none;
 }
+
+.ui.tabular.menu {
+    min-height: 40px;
```

Comment:
This is unnecessary

Reply:
Right, it calculates to `40.00000004` px now